src/styles/_catalog.scss

```
@@ -1,22 +1,34 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: $card-width;
+  grid-gap: 48px;
 
   &__card {
-    display: flex;
+    display: grid;
     justify-content: center;
     align-items: center;
-    width: 200px;
+    width: $card-width;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, $card-width);
+    }
```

```suggestion
    grid-template-columns: repeat(2, $card-width);
```

& is a `catalog` class so you can write it in shorter way